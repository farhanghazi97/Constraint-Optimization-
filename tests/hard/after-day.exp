m1:tue 9am
m2:wed 9am
m3:thu 9am
m4:fri 9am
cost:0
