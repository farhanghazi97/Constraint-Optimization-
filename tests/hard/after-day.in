meeting, m1
meeting, m2
meeting, m3
meeting, m4
domain, m1, after mon, hard
domain, m2, after tue, hard
domain, m3, after wed, hard
domain, m4, after thu, hard
