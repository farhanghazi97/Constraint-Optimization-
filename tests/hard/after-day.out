m1:fri 12pm
m2:fri 12pm
m3:fri 12pm
m4:fri 12pm
cost:0
